src/views/transaction-detail/index.js

```diff
@@ -81,7 +80,32 @@ class TransactionDetail extends Component {
 
     async getTx(txHash) {
         try {
-            const txResponse = await dataSource.getTx(txHash);
+          const txResponse = await dataSource
```

Comment:
Same old transformer for events... two places is fine, more than two places we can DRY it up